src/helpers/cardInfoUtil.js

```diff
@@ -52,12 +52,12 @@ class CardInfoUtil {
       },
       {
         title: BETTING,
-        description: `${getLocalDateTimeString(cOracle.startTime)} 
+        description: `${getLocalDateTimeString(cOracle.startTime)}
```

Comment:
does this next line work? I've had trouble where putting it on the next line didn't show the space that is needed.

Reply:
I didn't actually change this. I think it was my editor just removing spaces at the end of lines.